src/context/AppProvider/AppContext.tsx

```diff
@@ -168,6 +173,19 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
       dispatch(getValidatorData.initiate(accountId, options))
     }
 
+    dispatch(
+      getOpportunityMetadata.initiate(
+        {
+          // TODO: abstract me, we want to fire "eveything we need to fire" not an arbitrary opportunity data
```

Comment:
We obviously don't want to dispatch an action for each opportunity we want to fetch metadata for.
This works for now since we have a single opportunity we're fetching, but going forward, we will want to define an array of opportunities known ahead of time so we can map action dispatches